src/main/java/nextstep/courses/infrastructure/JdbcEnrollmentRepository.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.entity.EnrollmentEntity;
+import nextstep.courses.repository.EnrollmentRepository;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository("enrollmentRepository")
+public class JdbcEnrollmentRepository implements EnrollmentRepository {
```

Comment:
```suggestion
@Repository
public class JdbcEnrollmentRepository implements EnrollmentRepository {
```
이미 구현체를 하나만 가지고 있기 때문에 따로 명명해주지 않아도 괜찮을 것 같아요 😄